packages/dt_for_itables/src/index.js

```diff
@@ -29,6 +29,9 @@ import 'datatables.net-searchpanes-dt/css/searchPanes.dataTables.min.css';
 import 'datatables.net-select-dt';
 import 'datatables.net-select-dt/css/select.dataTables.min.css';
 
+// TODO: Uncaught ReferenceError: assignment to undeclared variable dt
+import 'datatables.net-plugins/dataRender/datetime.min.mjs';
+
```

Comment:
@AllanJard adding this import
```
import 'datatables.net-plugins/dataRender/datetime.min.mjs';
```
seems to break the package (causes an error _Uncaught ReferenceError: assignment to undeclared variable dt_ in the browser). Do you have an ES6 example for including the datetime extension? Thanks!

Reply:
There is no need for the date time rendering plugin any more. There is [one built into DataTables core](https://datatables.net/manual/data/renderers#Date-and-time-helpers).

That does currently need either moment.js or Luxon to be available on the `window`. With DT2.1 (still in development) I've added a way to pass Moment or Luxon to DataTables from a module rather than needing it in a global scope.

Reply:
Thanks Allan! I looked into a couple threads but I am afraid there is not much about es6 imports. When you say that Luxon should be available on the `window`, are you suggesting that something like
```
import {DateTime} from 'luxon';
window.DateTime = DateTime;
```
could work? (the above does not) Thanks!

Reply:
```
import luxon from 'luxon';
window.luxon = luxon;
```

I think would be needed to have DT2.0 work with Luxon in a module environment.

Reply:
I see. I have also tried to load Luxon using the below (and I've been able to use it in the console)
```
from IPython.display import HTML

HTML("""
<script src="https://cdn.jsdelivr.net/npm/luxon@3.4.4/build/global/luxon.min.js"></script>
""")
```
but that did not change the outcome for that sample table - so I am looking forward a module example for loading Luxon when you get a chance! Thanks!
```
<table id="itables_64e9588f_88d1_4fa1_86c8_a3ba4a8d2ee3" class="display nowrap" data-quarto-disable-processing="true" style="table-layout:auto;width:auto;margin:auto;caption-side:bottom">
<thead>
    <tr style="text-align: right;">
      
      <th>date</th>
    </tr>
  </thead><tbody><tr>
<td style="vertical-align:middle; text-align:left">
<div>
Loading ITables v2.1.2-dev from the internet...
(need <a href=[https://mwouts.github.io/itables/troubleshooting.html>help</a>?)</td](https://mwouts.github.io/itables/troubleshooting.html%3Ehelp%3C/a%3E?)%3C/td)>
</div>
</tr></tbody>

</table>
<link href="https://www.unpkg.com/dt_for_itables@2.0.10/dt_bundle.css" rel="stylesheet">
<script type="module">
    import {DataTable, jQuery as $} from 'https://www.unpkg.com/dt_for_itables@2.0.10/dt_bundle.js';

    document.querySelectorAll("#itables_64e9588f_88d1_4fa1_86c8_a3ba4a8d2ee3:not(.dataTable)").forEach(table => {
        // Define the table data
        const data = [["2023-04-19"]];

    // try DateTime (works!)
    console.log('datetime in itables', DateTime.now().setZone("America/New_York").minus({ weeks: 1 }).endOf("day").toISO());

        // Define the dt_args
        let dt_args = {"buttons": ["searchBuilder"], "layout": {"topStart": "buttons", "topEnd": null, "bottomStart": null, "bottomEnd": null}, "order": []};
        dt_args["data"] = data;

        
        new DataTable(table, dt_args);
    });
</script>
```

Reply:
Solved following the https://datatables.net/download/ page, which told me to do
```
npm install --save datatables.net-datetime
```
and to add
```
import DateTime from 'datatables.net-datetime';
```
to my local module - thanks Allan!